sklearn/tests/test_calibration.py

```diff
@@ -259,6 +259,17 @@ def test_calibration_curve():
     assert_raises(ValueError, calibration_curve, [1.1], [-0.1],
```

Comment:
Use an assertion like this (or assert_raise_message) to check for the error with incorrect strategy